src/compiler/factory.ts

```diff
@@ -2130,6 +2130,24 @@ namespace ts {
 
     // Compound nodes
 
+    export function createImmediatelyInvokedFunctionExpression(statements: Statement[]): CallExpression;
+    export function createImmediatelyInvokedFunctionExpression(statements: Statement[], param: ParameterDeclaration, paramValue: Expression): CallExpression;

```

Comment:
No, these are not arrays. As written, this function only allows zero or one parameter.